elanet/server.go

```diff
@@ -1003,3 +1024,8 @@ func makeEmptyMessage(cmd string) (p2p.Message, error) {
 	}
 	return message, nil
 }
+
+// nodeFlag returns if a peer contains the full node flag.
+func nodeFlag(flag uint64) bool {
+	return pact.ServiceFlag(flag)&pact.SFNodeNetwork != pact.SFNodeNetwork
```

Comment:
should be "pact.ServiceFlag(flag)&pact.SFNodeNetwork == pact.SFNodeNetwork"?

Reply:
Yeah, my mistake.